tools/state-viewer/src/epoch_info.rs

```diff
@@ -194,22 +207,30 @@ pub(crate) fn iterate_and_filter(
         .collect()
 }
 
+fn display_kickouts(epoch_info: &EpochInfo) {
+    for (key, value) in
```

Comment:
Using `(account_id, kickout_reason)` instead of `(key, value)` here would help with readability.